src/web/app/components/question-submission-form/question-submission-form.component.html

```diff
@@ -123,7 +123,7 @@
                               (modelChange)="triggerRecipientSubmissionFormChange(i, 'commentByGiver', $event)"
                               [isFeedbackParticipantComment]="true"
                               [questionShowResponsesTo]="model.showResponsesTo"
-                              [shouldHideSavingButton]="false"
+                              [shouldHideSavingButton]="true"
```

Comment:
The issue is more completed than you think (maybe). Saving comments requests are fired only if the button is clicked.

Which means that when `Submit` button is clicked, the associated comments will not be saved!

Reply:
Works fine for me though?

![output](https://user-images.githubusercontent.com/32454748/88623854-7143bf00-d0d8-11ea-9980-5955f3ad32bc.gif)

Reply:
Looks like I have considered the case when I write the code. :P

In this case, let's delete `updateParticipantComment` handler since it is not used